services/apps/alcs/src/queues/scheduler/application/status-emails/status-emails.consumer.ts

```diff
@@ -0,0 +1,107 @@
+import { Process, Processor } from '@nestjs/bull';
+import { Logger } from '@nestjs/common';
+import * as dayjs from 'dayjs';
+import * as timezone from 'dayjs/plugin/timezone';
+import * as utc from 'dayjs/plugin/utc';
+import { generateALCDApplicationHtml } from '../../../../../../../templates/emails/decision-released';
+import { generateREVAHtml } from '../../../../../../../templates/emails/under-review-by-alc.template';
+import { ApplicationSubmissionStatusService } from '../../../../alcs/application/application-submission-status/application-submission-status.service';
+import { SUBMISSION_STATUS } from '../../../../alcs/application/application-submission-status/submission-status.dto';
+import { ApplicationSubmissionToSubmissionStatus } from '../../../../alcs/application/application-submission-status/submission-status.entity';
+import { PARENT_TYPE } from '../../../../alcs/card/card-subtask/card-subtask.dto';
+import { EmailService } from '../../../../providers/email/email.service';
+import { QUEUES } from '../../scheduler.service';
+import { LocalGovernment } from '/Users/mekhti/Documents/Bit3/sources/alcs/services/apps/alcs/src/alcs/local-government/local-government.entity';
+import { ApplicationOwner } from '/Users/mekhti/Documents/Bit3/sources/alcs/services/apps/alcs/src/portal/application-submission/application-owner/application-owner.entity';
+import { ApplicationSubmission } from '/Users/mekhti/Documents/Bit3/sources/alcs/services/apps/alcs/src/portal/application-submission/application-submission.entity';
+
+dayjs.extend(utc);
+dayjs.extend(timezone);
+
+@Processor(QUEUES.APPLICATION_STATUS_EMAILS)
+export class ApplicationSubmissionStatusEmailConsumer {
+  private logger = new Logger(ApplicationSubmissionStatusEmailConsumer.name);
+
+  constructor(
+    private submissionStatusService: ApplicationSubmissionStatusService,
+    private emailService: EmailService,
+  ) {}
+
+  @Process()
+  async processSubmissionStatusesAndSendEmails() {
+    const today = dayjs(new Date())
```

Comment:
So this will send emails at 12:00 midnight our time?

Reply:
the very first ones yes and then the task will check every 15 minutes

Reply:
Discussed with Ansley, and this will be updated according to the new requirements: run daily every 15 minutes starting at 8 am